packages/autorest.typescript/test/smoke/generated/synapse-artifacts-rest/src/isUnexpected.ts

```diff
@@ -105,6 +105,7 @@ import {
   LibraryGetDefaultResponse,
   LibraryCreate200Response,
   LibraryCreate202Response,
+  LibraryAppend201Response,
```

Comment:
Why do we have these changes? I don't think we have sharedRoute cases in swagger?